Add note about size aggregate size, closes #50 (#51)
tremor-rs/tremor-www-docs: docs/tremor-query/functions/stats.md

The `stats` module contains functions for aggregating statistical measures
of various events.

## Size

> When using stats functions size becomes an important factor. The exact size an window using

Reviewer: ```suggestion
When using stats functions size becomes an important factor. The exact size a window using
```

Author: :+1:

> aggregates  occupies in memory depends on a two main factors:

Reviewer: ```suggestion
aggregates occupies in memory depends on two main factors:
```

* The size of the dimension identifier. I.e. if the window is identified by the string "window" it will
  require that amount of memory related to this. If it is identified by an array of 10.000 elements

>   all reading "window" it will use (about) 10.0000 that size.

Reviewer: ```suggestion
  all reading "window" it will use (about) 10.000 times that size.
```
